src/pages/content-edit/nav.js

```diff
@@ -157,10 +157,20 @@ module.exports = {
                     m("ul", { class : css.list },
                         current.items.map(function(data) {
                             var url = "/content/" + ctrl.schema.key + "/" + data.key,
-
-                                inactive = data.published ? "published" : "item";
-
-                            return m("li", { class : css[route.indexOf(url) === 0 ? "active" : inactive] },
+                                cssClass = css.item;
+
+                            if(data.published && route.indexOf(url) === 0) {
```

Comment:
This conditional makes me want to make an exception to my own rules & use nested ternaries :unamused: